apps/storybook-react/stories/WalletHome.stories.tsx

```diff
@@ -0,0 +1,358 @@
+import {
```

Comment:
We should be constantly updating this story with every new component that is applicable and be out benchmark for UI development